src/templates/home.tsx

```diff
@@ -34,20 +35,25 @@ interface Props {
 export default ({ data, pageContext }: Props) => (
   <Layout currentLocale={pageContext.language}>
     <BreadCrumbs crumbs={[{ text: <FormattedMessage id="HOME" /> }]} />
-
     <div className="container">
-      {data.categories.edges.map(({ node }) => (
-        <Link to={getCategoryPath(node)} key={node.id}>
-          <div className="card">
-            <div className="card-content">
-              <div
-                className="content"
-                dangerouslySetInnerHTML={{ __html: node.name }}
-              />
-            </div>
-          </div>
-        </Link>
-      ))}
+      {data.categories.edges.map(({ node }) => {
```

Comment:
Could do it like this, using `filter`, which is pretty nice:
```suggestion
      {data.categories.edges.filter(({ node }) => node.count > 0).map(({ node }) => (
        <Link to={getCategoryPath(node)} key={node.id}>
```